traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -312,6 +312,9 @@ sub param_data {
 		if ( $row->parameter->name eq "location" ) {
```

Comment:
yes, it feels like the get_scope function should suffice and then it gets rid of the need to create scope parameters... and figure out which profiles to attach them to... and attach them to profiles... and create new scope params when delivery services are updated....and whatever else is needed...

i know it's not quite as "flexible" but man, really seems to simplify things...